docs/modules/table.md

```diff
@@ -42,16 +42,16 @@ insertRow(offset: number)
 **Examples**
 
 ```js
-quill.table.insertRow(1); // insert row below
-quill.table.insertRow(0); // insert row above
-quill.table.insertRow(-3); // insert a row three rows above the current position
-quill.table.insertRow(3); // insert a row three rows below the current position
+quill.table.insertRow(1); // inserts row below
+quill.table.insertRow(0); // inserts row above
+quill.table.insertRow(-3); // inserts a row above 3 rows from the current position
+quill.table.insertRow(3); // inserts a row below 2 rows from the current position
```

Comment:
The commands states 3, but the text says 2. So which is it? I use 3 in my suggestion.

```suggestion
quill.table.insertRow(3); // inserts a row 3 rows below the current position
```